test/box/alter.result

```diff
@@ -141,15 +160,20 @@ space:replace{0}
 ...
 _index:insert{_space.id, 0, 'primary', 'tree', {unique=true}, {{0, 'unsigned'}}}
 ---
-- error: Duplicate key exists in unique index 'primary' in space '_index'
+- error: 'Duplicate key exists in unique index ''primary'' in space ''_index'': old
+    tuple - ''[280, 0, "primary", "tree", {"unique": true}, [[0, "unsigned"]]]'',
```

Comment:
Two quota signs before tuple seems to be strange: ''[
Let's remove at least one from commit message format.

Reply:
Also you didn't add quotation signs to index/space name, but now there's two of them. Why?

Reply:
Meanwhile format in sql-tap tests hasn't changed. Looks strange. Could you please dig into this issue and explain it?

Reply:
I think in one case the string is in `'` and in the other in `"`. Because of this, they are escaped in different ways - if the string is in  `'`, then  `'` is added, and if it is in `"`, then `\` is added.
I've experimented with different quotes and I think I've found a suitable one: adding escaping for the case of `"`.